Northstar.CustomServers/mod/scripts/vscripts/mp/_base_gametype_mp.gnut

```diff
@@ -628,6 +628,10 @@ void function SetRecalculateRespawnAsTitanStartPointCallback( entity functionref
 
 bool function ShouldEntTakeDamage_SPMP( entity ent, var damageInfo )
 {
+	// dropships are immune to being crushed
+	if ( ( IsDropship( ent ) || IsEvacDropship( ent ) ) && DamageInfo_GetCustomDamageType( damageInfo ) & DF_TITAN_STEP )
```

Comment:
```suggestion
	if ( ( IsDropship( ent ) || IsEvacDropship( ent ) ) && IsTitanCrushDamage( damageInfo ) )
```
`IsTitanCrushDamage` is a builtin function to do such checks, tested in both ways and both works, but i suggest using the builtin function check for this.

Reply:
Looking at it, yeah you are right. However, I think that there is a bug in `IsTitanCrushDamage` as well. It checks that `DamageInfo_GetCustomDamageType( damageInfo ) == damageTypes.titanStepCrush` when `DamageInfo_GetCustomDamageType( damageInfo )` is a set of bitflags. An example of when this check would fail is if the damage type was set to `DF_TITAN_STEP + DF_DOOMED_HEALTH_LOSS`